trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trial_config_service/config/WebConfig.java

```diff
@@ -1,80 +1,17 @@
 package uk.ac.ox.ndph.mts.trial_config_service.config;
 
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.cors.CorsConfiguration;
 import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 import org.springframework.web.filter.CorsFilter;
 import org.springframework.web.reactive.function.client.WebClient;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
-import springfox.documentation.builders.*;
-import springfox.documentation.service.*;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spi.service.contexts.SecurityContext;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger.web.SecurityConfiguration;
-import springfox.documentation.swagger.web.SecurityConfigurationBuilder;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import static java.util.Arrays.asList;
 
 @Configuration
-public class WebConfig implements WebMvcConfigurer, WebConfigService {
-
-    private static final String API_MTS_APPS_ADMIN = "api://mts-apps/admin";
-    private static final String ADMIN_SCOPE = "admin scope";
-
-    // TODO ARTS-264 put these values into Azure Key Vault
-    @Value("${spring.security.oauth2.resourceserver.opaquetoken.client-secret}")
-    private String clientSecret;
-
-    // TODO ARTS-264 put these values into Azure Key Vault
-    @Value("${spring.security.oauth2.resourceserver.opaquetoken.client-id}")
-    private String clientId;
+public class WebConfig implements WebMvcConfigurer {
 
-    // TODO ARTS-264 put these values into Azure Key Vault
-    @Value("${swagger.authserver.url}")
-    private String authServer;
-
-    private List<AuthorizationScope> authorizationScopeList;
-
-    WebConfig() {
-        authorizationScopeList = new ArrayList<>();
-    }
-
-    @Bean
-    public Docket api() {
```

Comment:
This is still required for swagger, please put it back in

Reply:
Hi Saj - readded removed Swagger config